test/integration/component/test_blocker_bugs.py

```diff
@@ -663,7 +663,14 @@ def test_01_restart_network_cleanup(self):
         #    all it's services should resume
 
         # Find router associated with user account
-        list_router_response = list_routers(
```

Comment:
Since list  router vm is getting called multiple time based on zone networking type  , can we use a function which list router vm based on networking type .
Please ignore the comment if you feel it is not required .